src/component/reducer/actions/LoadActions.ts

```diff
@@ -46,10 +46,36 @@ function setData(
       tolerance: DefaultTolerance,
     });
   } else {
-    draft.correlations = correlations;
+    // in case of older NMRium data are imported, convert hybridization string to number array
+    // @TODO remove following command to overwrite correlations at some point in future
+    draft.correlations =
```

Comment:
For me we don't need to support a migration. This was an experimental feature so it may be breaking.